plugin.js

```diff
@@ -0,0 +1,50 @@
+const AdminBro = require('admin-bro')
+
+module.exports = {
+  name: 'AdminBroExpressjs',
+  version: '0.1.0',
+
+  register: async (app, options) => {
+    const express = require('express')
+    const router = express.Router()
+    const path = require('path');
+    const bodyParser = require('body-parser');
+
+    const { routes, assets } = AdminBro.Router
+    const admin = new AdminBro(options)
+
+    app.use(bodyParser.json())
```

Comment:
I think it should be `router.use` instead of `app.use`, and thus - app is not even needed, so you could skip this param. 

When you invoke app.use you changing all routes (even those which arent in `/admin` scope)